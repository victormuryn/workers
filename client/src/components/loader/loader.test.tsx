import React from 'react';
import renderer from 'react-test-renderer';
import Loader from './loader';

describe(`Loader tests`, () => {
  it(`Loader renders corrects`, () => {
    const tree = renderer.create(<Loader />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
