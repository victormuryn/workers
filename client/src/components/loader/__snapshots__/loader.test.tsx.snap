// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader tests Loader renders corrects 1`] = `
<div
  className="spinner__wrapper"
>
  <div
    className="spinner"
  >
    <div
      className="visually-hidden"
    >
      Триває загрузка...
    </div>
    <div
      className="inner one"
    />
    <div
      className="inner two"
    />
    <div
      className="inner three"
    />
  </div>
</div>
`;
